### PR TITLE
Added custom giphy request to annoy AndyAllan

### PR DESCRIPTION
**Pull request by coldfumonkeh**
Watches for specific giphy terms to display a SFW gif image

### CODE REVIEW

File: plugins/giphy.js [hunk 1]
```diff
@@ -13,13 +13,18 @@ module.exports = (function(){
...
 			if (text.indexOf('giphy:') === 0 && text.length >= 5) {
-				randomGiphy(text.substr(6), function(err, url){
-					if (err){
-						bot.say(to, err);
-					}else{
-						bot.say(to, url);
-					}
-				});
+				var term = text.substr(6);
+				if(term === 'reacharound' || 'reachie' || 'reach-around') {
```
Comment (atuttle) [inline plugins/giphy.js]: That's going to return `true` if term ==='reacharound`, else it'll return `'reachie'` thus the condition will always evaluate to true. I don't think that's what you intended...

Comment (ryanguill) [inline plugins/giphy.js]: If you pull in lodash (```var _ = require('lodash')```) you can do ``` if ( _.contains(['reacharound', 'reachie', 'reach-around'], term))```

**Comment by coldfumonkeh** [comment]
Thanks, revised with the lodash method. Testing in monkehbot PM

**Comment by coldfumonkeh** [comment]
New Spotify plugin added to translate spotify URIs into readable text with links to open in browser. Quite useful as you know what the URIs are instead of clicking blind.

**Comment by atuttle** [comment]
BTW in the future please put each feature into a separate branch in your fork so that they can be handled in separate pull requests.

**Comment by coldfumonkeh** [comment]
Will do, thanks Adam

**Pull request merged**